src/main/java/com/rackspace/salus/monitor_management/web/model/ValidationGroups.java

```diff
@@ -33,5 +33,6 @@
   public interface Create extends Default { }
   public interface Update extends Default { }
   public interface Patch extends Default { }
+  public interface TranslateMonitorContent extends Default { }
```

Comment:
I'm a little torn between using a validation group for this vs. creating a new class with only the fields we care about.  

In either case I'm struggling to come up with good names.  If we use the group I think it should be a more specific operation like `TranslationWithoutMonitorProvided` since both uses of `TranslateMonitorContentRequest` are to translate monitor content. We'd want to be able to easily distinguish between the use-cases.

```suggestion
  public interface TranslationWithoutMonitorProvided extends Default { }
```